src/test/cli/rbd/help.t

```diff
@@ -28,8 +28,8 @@
       group remove (group rm)     Delete a consistency group.
       image-meta get              Image metadata get the value associated with
                                   the key.
-      image-meta list             Image metadata list keys with values.
-      image-meta remove           Image metadata remove the key and value
+      image-meta list(image-meta ls)   Image metadata list keys with values.
+      image-meta remove(image-meta rm) Image metadata remove the key and value
```

Comment:
once i put it aligened with context, it will say "This branch has conflicts that must be resolved" ,which does not happend if i just change those two line. i do not know why